### ISSUE TITLE
508-defect-2 [SCREENREADER]: On this page should be a navigation landmark following VA design system guidance

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#28349]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Details

On this page should be a unique navigational landmark according to VA design system guidance. This would help screen reader users quickly find and access the links within "On this page."

## Acceptance Criteria

- [x] [Follow VA design system guidance for "On this page" component](https://design.va.gov/components/on-this-page-jump-links)

## Steps to Recreate

1. Log in as user 1 and go to `https://staging.va.gov/manage-va-debt/your-debt/#currentDebts`
2. Using this [landmarks Chrome extension](https://chrome.google.com/webstore/detail/landmark-navigation-via-k/ddpokpbjopmeeiiolheejjpkonlkklgp?hl=en-US), confirm "On this page" does not emerge as a landmark

## WCAG or Vendor Guidance (optional)

[W3C Navigation Landmarks](https://www.w3.org/TR/wai-aria-practices/examples/landmarks/navigation.html)

## Screenshots or Trace Logs

<img width="1352" alt="Screen Shot 2021-08-05 at 1 28 22 PM" src="https://user-images.githubusercontent.com/14154792/128394692-bffb0c49-1f63-4106-925f-94a8a84d3760.png">

**Comment by denisecoveyduc** [comment]
This issue will be resolved as part of #53552

**Comment by HeatherRienks** [comment]
@OptionSelect to confirm if this has been completed.

**Comment by OptionSelect** [comment]
OnThisPageLinks.jsx and OnThisPageOverview.jsx need updates (I believe wrapping with Nav component) to properly have the H2 registered as a landmark on the page.

https://www.w3.org/WAI/ARIA/apg/patterns/landmarks/examples/navigation.html

<img width="773" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMTBZQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--87798646d2e48e01571a02e0a97e982d201cc4b3/image.png" alt="image.png" />

<img width="966" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMTRZQXc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cd9b19faa1a390703fa9829e35c1b0908db2b92c/image.png" alt="image.png" />

**Comment by josephrlee** [comment]
@coforma-terry this has been [completed with this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/27832), can you please validate and close it out? Thanks!

**Comment by HeatherRienks** [comment]
Per @josephrlee - this has been completed. He will confirm with Terry Nichols

**Comment by josephrlee** [comment]
@coforma-terry, just wanted to pint you again on this before I close this out! Thank you!

**Comment by eli-oat** [comment]
Nailed it @josephrlee! This looks great to me. 

For continuity I've attached an image showing that the landmark is no longer appearing ✅ 

<img width="1332" alt="Screenshot of the form in staging, with a pagestructure outline in the left." src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/3342274/f44bde36-bf55-482b-996a-daa5be5f8556">

**Comment by josephrlee** [comment]
@HeatherRienks , this ticket has been validated by CAIA and has been closed out! 🥳
